src/app/entries/entries-list/entries-list.component.html

```diff
@@ -5,6 +5,11 @@
       <h1 class="page-header-title">
         {{ 'strings.entityTravelEntries' | translate }}
       </h1>
+      <div class="page-header-actions">
+        <button class="add-btn" mat-flat-button color="primary" (click)="openAddTravelEntryModal()">
+          {{ 'strings.headingCreateNewTravelEntry' | translate }}
```

Comment:
this string is for headings, not buttons
there already should be a translation for that

Reply:
Resolved.